code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -421,8 +421,8 @@
 	if(M.a_intent == INTENT_HELP)
 		playsound(loc, 'sound/effects/glassknock.ogg', 25, 1)
 		M.visible_message(SPAN_WARNING("[M] creepily taps on [src] with its huge claw."), \
-		SPAN_WARNING("You creepily tap on [src]."), \
-		SPAN_WARNING("You hear a glass tapping sound."), 5, CHAT_TYPE_XENO_COMBAT)
+		SPAN_WARNING("We creepily tap on [src]."), \
+		SPAN_WARNING("We hear a glass tapping sound."), 5, CHAT_TYPE_XENO_COMBAT)
```

Comment:
The 3rd one is an audio message that will be displayed to anyone blind. This is pretty much only non-xenos since they only go blind on crit